docs/data/install-xrootd.md

```diff
@@ -425,7 +444,69 @@ root@host # su - %RED%user%ENDCOLOR%
 \[xrootd\] Total 0.00 MB |**`================`**| 100.00 % \[inf MB/s\]
 ```
 
-#### Security option 2: xrootd-lcmaps authorization
+#### Security option 2: Shared keys
+
+If you want to enable security for access to xrootd via xrootdfs you will need to modify xrootd configuration and perform several steps to make xrootdfs secured.
+
+1. On the xrootd redirector node, execute the following command:
+
+        :::console
+        root@host # xrdsssadmin -k %RED%<my_key_name>%ENDCOLOR% -u anybody -g usrgroup add %RED%<keyfile>%ENDCOLOR%
+
+    eg:
+
+        root@host # xrdsssadmin -k top_secret -u anybody -g usrgroup add /etc/xrootd/xrootd.key
+
+2. Set ownership
+
+        :::console
+        root@host # chown xrootd.xrootd /etc/xrootd/xrootd.key
+
+3. On the node where xrootdfs is installed modify **`/etc/fstab`** add security information:
+
+        :::console
+        root@host # xrootdfs %RED%/mnt/xrootd %ENDCOLOR% fuse rdr=xroot://%RED%redirector1.domain.com%ENDCOLOR%:1094/%RED%/path/redirector1%ENDCOLOR%,uid=xrootd,sss=%RED%keyfile%ENDCOLOR%0 0
+
+4. On all xrootd data servers and redirector nodes, modify xrootd configuration (**`/etc/xrootd/xrootd-clustered.cfg`**) by adding the following segment:
+
+        :::file
+        # ENABLE_SECURITY_BEGIN
+           xrootd.seclib /usr/lib64/libXrdSec.so
+           #the line below should be before "sec.protocol ... unix"
+           %RED%sec.protocol /usr/lib64 sss -s keyfile %ENDCOLOR%
+           sec.protocol /usr/lib64 unix
+           # this specify that we use the 'unix' authentication module, additional one can be specified.
+           # this is the authorization file
+           acc.authdb /etc/xrootd/auth_file
+           ofs.authorize
+           # ENABLE_SECURITY_END
+
+5. On all xrootd data server nodes, edit `/etc/xrootd/auth_file` to add authorized users of the form `u %RED%username%ENDCOLOR% %RED%/directoryname%ENDCOLOR% lr` where "lr" is the permission set.
+
+6. Copy %RED%keyfile<span class="twiki-macro ENDCOLOR"></span> from redirector node to every data server node and the xrootdfs node. Make sure that this file is owned by the `xrootd` user.
+
+7. Restart xrootd cluster by following [these instructions](https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/HowToInstallXrootd)
```

Comment:
Oof, twiki link and no indication of which instructions on the linked page should be followed

Reply:
Done.  I simplified this just to "restart the daemons".